interop/idl/interop.bond

```diff
@@ -26,4 +26,7 @@ struct OneOfEverything {
     16: map<int32, string> m_i_str;
     17: SomeEnum e = A;
     18: OneOfSomething oos;
+    19: blob blorb;
+    20: nullable<int32> n_nothing;
```

Comment:
Please use another name, as this doesn't default to "nothing" and could easily be misinterpreted. Perhaps n_null and n_not_null?